source/INI/PeanutButter.INI/INIFile.cs

```diff
@@ -437,6 +446,25 @@ public void RemoveSection(string section)
             Data.Remove(section);
         }
 
+        public void RenameSection(string existingName, string newName)

```

Comment:
implementation looks fine to me -- please just update the test to be as pedantic as possible (:

Reply:
ok